system-test/logging.js

```diff
@@ -281,7 +281,7 @@ describe('Logging', () => {
   });
 
   describe('logs', () => {
-    const log = logging.log('syslog');
+    const log = logging.log('systestlog');
```

Comment:
Can you use a uuid in the string here to make sure two concurrent runs of the tests don't interfere? (yes this has happened!)

Reply:
Good call! Done